app/api/contact/specs/contact.spec.js

```diff
@@ -36,5 +37,30 @@ describe('contact', () => {
         })
         .catch(catchErrors(done));
     });
+    it('should send email with the provided sender email and site name', async done => {
+      const settings = {
+        site_name: 'some site name',
+        senderEmail: 'sender@email.com',
+      };
+      const set = await settingsModel.get();
+      const updated = Object.assign(set[0], { ...settings });
+      await settingsModel.save(updated);
```

Comment:
using settings.ts instead of settingsModel will allow you to save the changes directly without requesting anything to db etc, this way we will have a more concise test.

```
await settings.save({ site_name: 'some site name',senderEmail: 'sender@email.com'})